spec.html

```diff
@@ -6211,6 +6222,8 @@
         1. Add _pending_ at the back of the Job Queue named by _queueName_.
         1. Return NormalCompletion(~empty~).
       </emu-alg>
+
+      <emu-note>For host environments consisting of multiple continents, when those host environments enqueue their own jobs, they should generally specify which continent's job queues they are enqueuing on.</emu-note>
```

Comment:
> I want it to be obvious where EnqueueJob("ScriptJobs", ...) is actually happening

Maybe it's obvious already? EnqueueJob calling some other Continent doesn't seem a likely implementation alternative...